src/api-service/__app__/agent_events/__init__.py

```diff
@@ -125,6 +126,8 @@ def on_state_update(
                             machine_id,
                             done_data,
                         )
+        else:
+            logging.info("No change in Node state")
```

Comment:
This needs to include information about the node in question.  Given this is about a NOOP, it can also be made a `debug`